tokens/0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874.yaml

```diff
@@ -0,0 +1,20 @@
+---
+addr: '0xd1ec43b9d44338a4e74f3f41cd1b86254cbaa874'
+decimals: 18
+description: >- Tokenized 1 Troy Ounce Gold Bar, Redeemable in Precious Metal (Gold)
+  # The following options were found                                                                       #
+  # You may need to edit them and you MUST delete commented lines                                          #
+  # From the website: description meta tag                                                                 #
+  Veritaseum enables software-driven P2P capital markets without brokerages, banks or traditional exchanges
+links:
+- Facebook: https://www.facebook.com/reggiemiddletonfintech/?fref=ts
+- Telegram: https://t.me/joinchat/G_22OhA7UgN6b-7hTSFSTA
+- Twitter: https://twitter.com/reggiemiddleton
+- Website: https://veritas.veritaseum.com/
+- YouTube: https://www.youtube.com/channel/UCjVEF7ABJwyzw_8pb5frVmg
+- Blog: https://veritas.veritaseum.com/ve-blog
+- Reddit: https://www.reddit.com/r/VeritaseumCommunity/
+
+
```

Comment:
Extra white space here should be deleted.

Reply:
Extra lines removed